src/new-client/src/Web/serviceWorkerRegistration.ts

```diff
@@ -28,6 +28,9 @@ type Config = {
 export function register(config?: Config) {
   if ("serviceWorker" in navigator) {
     window.addEventListener("load", () => {
+      // This url currently breaks when running against the env gateway as it misses the correct pathname
+      // e.g. web.env.reactivetrader.com/sw.js instead of web.env.reactivetrader.com/branch/hydra-integration/sw.js
```

Comment:
ah, this must be something to do with the url path modifer in nginx?

Reply:
it does indeed, the correct url would be the origin + the full pathname for the sw.js to be downloaded correctly, however we are still getting issues with that. I want to address this as a separate issue as it only affects the env environments and the solution is not trivial

Reply:
but I think the _correct_ URL is just `origin/sw.js` and the lb/nginx should be sorting out your URL rewriting ..

Reply:
I can see no evidence of this being applied in nginx or elsewhere - my comment was in relation to a conversation with Gary a couple of weeks back, so perhaps that was reverted or maybe never properly applied .. oh well.